AnnouncementsEdit.ascx.cs

```diff
@@ -198,20 +199,29 @@ private void BindForm()
                     if ((!Null.IsNull(Model.AnnouncementInfo.PublishDate)) &&
                         (Model.AnnouncementInfo.PublishDate != (DateTime)SqlDateTime.Null))
                     {
-                        publishDate.SelectedDate = Model.AnnouncementInfo.PublishDate;
-                        publishTime.SelectedDate = Model.AnnouncementInfo.PublishDate;
+                        var portalDatetime = TimeZoneInfo.ConvertTimeFromUtc(Model.AnnouncementInfo.PublishDate.Value, ModuleContext.PortalSettings.TimeZone);
+                        publishDate.SelectedDate = portalDatetime;
+                        publishTime.SelectedDate = portalDatetime;
                     }
                     if ((!Null.IsNull(Model.AnnouncementInfo.ExpireDate)) &&
                         (Model.AnnouncementInfo.ExpireDate != (DateTime)SqlDateTime.Null))
                     {
-                        expireDate.SelectedDate = Model.AnnouncementInfo.ExpireDate;
-                        expireTime.SelectedDate = Model.AnnouncementInfo.ExpireDate;
+                        var portalDateTime = TimeZoneInfo.ConvertTimeFromUtc(Model.AnnouncementInfo.ExpireDate.Value, ModuleContext.PortalSettings.TimeZone);
+                        expireDate.SelectedDate = portalDateTime;
+                        expireTime.SelectedDate = portalDateTime;
                     }
 
-                    ctlAudit.CreatedDate = Model.AnnouncementInfo.CreatedOnDate.ToString(CultureInfo.InvariantCulture);
+                    var user = UserController.Instance.GetCurrentUserInfo();
+                    var userPreferredCulture = CultureInfo.InvariantCulture;
+                    if (user != null && !string.IsNullOrWhiteSpace(user.Profile.PreferredLocale))
+                    {
+                        userPreferredCulture = new CultureInfo(user.Profile.PreferredLocale);
+                    }
+                    
+                    ctlAudit.CreatedDate = TimeZoneInfo.ConvertTimeFromUtc(Model.AnnouncementInfo.CreatedOnDate, ModuleContext.PortalSettings.TimeZone).ToString(userPreferredCulture);
```

Comment:
I think this should be utc or portal time for the audit

Reply:
Yep, it is in the portal preferred timezone but formatted in the current user culture, makes sense or I misenterpreted your comment ?

Reply:
My bad, looks good